irods/test/pool_test.py

```diff
@@ -237,6 +241,73 @@ def test_no_refresh_connection(self):
             self.assertEqual(0, len(self.sess.pool.active))
             self.assertEqual(0, len(self.sess.pool.idle))
 
+    def test_connection_destructor_called(self):
```

Comment:
Test to confirm the connection destructor log message is actually logged to file (to confirm the destructor is called).